src/proc/formats-converter.cpp

```diff
@@ -57,38 +57,34 @@ void formats_converter::drop_non_basic_formats()
     }
 }
 
-std::string to_string( std::shared_ptr< stream_profile_interface > profile )
+std::ostream & operator<<( std::ostream & os, const std::shared_ptr< stream_profile_interface > & profile )
```

Comment:
Can you please add the required include on top?
So we will not be depending on other headers?

Reply:
Added